jellyfin_mpv_shim/player.py

```diff
@@ -124,6 +124,7 @@ class to prevent concurrent control events hitting the player, which
     def __init__(self):
         self._video = None
         mpv_options = OrderedDict()
+        checkMPVLocation = settings.mpv_ext_path if settings.mpv_ext_path != None and platform.system() != "Darwin" else "../Resources/mpv"
```

Comment:
`settings.mpv_ext_path != None and platform.system() != "Darwin"`
Should be
`settings.mpv_ext_path is not None or platform.system() != "Darwin"`

Reply:
So should I add an additional check for whether it's frozen or not on `checkMPVLocation`?

Reply:
Yes

Reply:
Before I commit, take a look at this:
`packaged_mpv = "../Resources/mpv"
  checkMPVLocation = settings.mpv_ext_path if settings.mpv_ext_path is not None and \
            platform.system() != "Darwin" and getattr(packaged_mpv, 'frozen', False) else packaged_mpv`

Reply:
If `settings.mpv_ext_path` is not None it should always be used. If it is None, then packaged_mpv should be used as long as both `platform.system() == "Darwin" and getattr(sys, 'frozen', False)`.

Reply:
Gotcha. I've updated accordingly.

Reply:
```
mpv_location = settings.mpv_ext_path
# Use bundled path for MPV if not specified by user, on Mac OS, and frozen
if mpv_location is None and platform.system() == "Darwin" and getattr(sys, 'frozen', False):
     mpv_location = "../Resources/mpv"
```